docs/GettingStarted.md

```diff
@@ -387,6 +387,38 @@ giving precedence to the middleware settings. Inherited configurations are:
 * ``trace_agent_hostname``
 * ``trace_agent_port``
 
+### Faraday
+
+The `faraday` integration is available through the `ddtrace` middleware:
+
+    require 'faraday'
+    require 'ddtrace'
+
+    Datadog::Monkey.patch_module(:faraday) # this will simply register our middleware
```

Comment:
I'd say to remove `simply`: "adds the tracing middleware" or something similar

Reply:
👍